3rdParty/iresearch/core/utils/bitvector.hpp

```diff
@@ -305,6 +317,16 @@ class bitvector final {
 
   size_t words() const NOEXCEPT { return set_.words(); }
 
+  template<typename Visitor>
+  bool visit(Visitor visitor) const {
```

Comment:
would it make sense to not copy the visitor here, i.e. can you change to signature to a `Visitor const&`?